DigitalLearningSolutions.Web/Views/TrackingSystem/Delegates/ViewDelegate/ConfirmRemoveFromCourse.cshtml

```diff
@@ -0,0 +1,43 @@
+﻿@inject IConfiguration Configuration
+@using DigitalLearningSolutions.Web.Models.Enums
+@using DigitalLearningSolutions.Web.ViewModels.TrackingSystem.Delegates.ViewDelegate
+@using Microsoft.Extensions.Configuration
+@model RemoveFromCourseViewModel
+
+@{
+  var errorHasOccurred = !ViewData.ModelState.IsValid;
+  ViewData["Title"] = $"Remove delegate enrolment from {Model.CourseName}";

```

Comment:
In other pages, the tab title has "Error: " prefixed to it when we display an error summary. This page should do this as well.